src/Avalonia.Controls/Primitives/Popup.cs

```diff
@@ -94,31 +92,26 @@ static Popup()
         {
             IsHitTestVisibleProperty.OverrideDefaultValue<Popup>(false);
             ChildProperty.Changed.AddClassHandler<Popup>((x, e) => x.ChildChanged(e));
-            IsOpenProperty.Changed.AddClassHandler<Popup>((x, e) => x.IsOpenChanged(e));
-        }
-
-        public Popup()
-        {
-            
+            IsOpenProperty.Changed.AddClassHandler<Popup>((x, e) => x.IsOpenChanged((AvaloniaPropertyChangedEventArgs<bool>)e));
```

Comment:
We should probably add an overload which does this for you now we have strongly-typed property changed event args (probably in another PR?).

Reply:
Yeah, I thought about this as well - but indeed it is out of scope for this PR.